src/luarocks/cmd.tl

```diff
@@ -595,27 +623,27 @@ function cmd.run_command(description, commands, external_namespace, ...)
 
    local args, cmdline_vars = process_cmdline_vars(...)
    local parser = get_parser(description, cmd_modules)
-   args = parser:parse(args)
+   local argsp: Args = parser:parse(args) as Args --!
```

Comment:
instead of naming this `argsp`, rename the other one into `cmdline_args` (because this "argsp" is the same object that appears as "args" in every other cmd module)